gspread/models.py

```diff
@@ -12,8 +12,12 @@
 from collections import defaultdict
 from itertools import chain
 
-from xml.etree import ElementTree
-from xml.etree.ElementTree import Element, SubElement
+try:
+    from lxml.etree import ElementTree
+    from lxml.etree.ElementTree import Element, SubElement
```

Comment:
This import raises `ImportError` on my system (Python 2.7.10, lxml 3.4.4):

~~~
>>> from lxml.etree.ElementTree import Element, SubElement
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named ElementTree

>>> import lxml
>>> lxml.etree.__version__
u'3.4.4'

~~~

So for this module the `xml.etree` version is imported in the `except` clause.

The [compatibility guide](http://lxml.de/compatibility.html) suggests this way of import:

~~~
from lxml.etree import Element
~~~